### PR TITLE
add style to description__text to enable multiline

### PR DESCRIPTION
**Pull request by Ge0**

### CODE REVIEW

**Comment by Ge0** [comment]
@just-boris This should be fine by now, what do you think?

File: allure-report-face/src/styles.css [hunk 1]
```diff
@@ -66,6 +66,11 @@ html, body {
   white-space: nowrap;
 }
 
+.description__text {
```
Comment (just-boris) [inline allure-report-face/src/styles.css]: this class should be named `preformatted-text`.
We are strictly following BEM naming convention, so this file is supposed to contain only class names with general functionality.

**Comment by just-boris** [comment]
That's better.
But in the second version, we have used a different class name, and this is important because our classes are also part of API for plugins.

Could you change the class name in CSS and also add the new class name to `DescriptionView.hbs` file as well.

**Comment by Ge0** [comment]
@just-boris thanks for your feedback, I just pushed a new commit that changes the class name. Tell me if you actually wanted me to meld the two commits into one.

**Comment by Ge0** [comment]
How is the merge process working on this project?

**Comment by baev** [comment]
@Ge0 nothing special

**Comment by baev** [comment]
retest this please

**Comment by baev** [comment]
retest this please

**Comment by baev** [comment]
@Ge0 thanks!

**Pull request merged**